### ISSUE TITLE
CreateCards and CreateCard() raise error.

### ISSUE THREAD

**Issue by henryhu123** [Merisho/tx-holdem#7]
```
const { Pack, Hand, HandsCollection } = require('tx-holdem');

const pack = new Pack();

const board = new Hand(pack.createCards(5));

const pocket = new Hand([
    pack.createCard(),
    pack.createCard(),
]);

const coll = HandsCollection.createCombinations(board, pocket);

console.log('Highest is', coll.highestCombination.name);
```

or

```
const { Pack, Hand, HandsCollection } = require('tx-holdem');

const pack = new Pack();

const board = new Hand([
    pack.createCard(),
    pack.createCard(),
    pack.createCard(),
    pack.createCard(),
    pack.createCard(),
]);

const pocket = new Hand([
    pack.createCard(),
    pack.createCard(),
]);

const coll = HandsCollection.createCombinations(board, pocket);

console.log('Highest is', coll.highestCombination.name);
```

Both shows error in console with following information,

```
        this.cards.sort((l, r) => order === 'asc' ? l.compare(r) : r.compare(l));
                                                      ^

TypeError: Cannot read property 'compare' of null
    at cards.sort (C:\Users\HU\Desktop\c09\node_modules\tx-holdem\src\hand.js:220:55)
    at Array.sort (native)
    at Hand.sort (C:\Users\HU\Desktop\c09\node_modules\tx-holdem\src\hand.js:220:20)
    at new Hand (C:\Users\HU\Desktop\c09\node_modules\tx-holdem\src\hand.js:20:14)
    at Object.<anonymous> (C:\Users\HU\Desktop\c09\app.js:15:15)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

Did I miss something or it is a bug?

**Comment by OleksiiTorunov-dataart** [comment]
Hi @henryhu123 ,

thanks for your request. Will take a look at it today

**Comment by Merisho** [comment]
Hi @henryhu123 ,

thanks for your request. I will take a loot at it today

**Comment by Merisho** [comment]
Hello @henryhu123 ,

The issue has been resolved in the new 1.0.0 version. Please try to use it again.
Thank you for your issue request.
